config/install/field.storage.user.field_gender.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - user
+id: user.field_gender
+field_name: field_gender
+entity_type: user
+type: list_string
+settings:
+  allowed_values:
```

Comment:
At these days and age, there will be probably more options :)
Anyway, let's clarify with Zohar.